src/sagemaker/serve/builder/jumpstart_builder.py

```diff
@@ -469,3 +474,10 @@ def _build_for_jumpstart(self):
             )
 
         return self.pysdk_model
+
+    def _is_gated_model(self, model) -> bool:
+        """Determine if ``this`` Model is Gated"""
```

Comment:
nit: can we start having docstrings as the following from now on:
    """Checks if the provided model is gated.

    Args:
        model: The model object.
    Returns:
        bool: True if is gated model.
    """
    
   You might also want to have some type hints, ie. model: Model